gptcache/adapter/langchain_llms.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional, List
+from .adapter import adapt
+
+
+class LangChinLLMs:
+    _llm = None
+    def __init__(self, llm):
+        if self._llm is not None:
+            raise RuntimeError('llm has been created')
```

Comment:
When Will the __init__ method be called except when the object is created?

Reply:
Just to prevent users from creating duplicate

Reply:
fixed